src/handle_wrap.cc

```diff
@@ -38,11 +38,11 @@ void HandleWrap::Unref(const FunctionCallbackInfo<Value>& args) {
 }
 
 
-void HandleWrap::IsRefed(const FunctionCallbackInfo<Value>& args) {
+void HandleWrap::Unrefed(const FunctionCallbackInfo<Value>& args) {
   HandleWrap* wrap = Unwrap<HandleWrap>(args.Holder());
 
-  bool refed = IsAlive(wrap) && (wrap->flags_ & kUnref) == 0;
```

Comment:
There's no message body explaining why you removed the `IsAlive()` check, so i'm not sure of your rational. Seeing checks removed without explanation make me quiver.